teams/icebreakers-llvm.toml

```diff
@@ -7,4 +7,5 @@ members = [
     "nagisa",
     "nikic",
     "rkruppe",
+    "heyrutvik"
```

Comment:
You can add yourself at the top of the list to keep it sorted.

Reply:
Done. Thanks!